spec/rubocop/cop/style/redundant_line_continuation_spec.rb

```diff
@@ -150,6 +150,13 @@ def self.foo(bar,#{trailing_whitespace}
     RUBY
   end
 
+  it 'does not register an offense when using string concatenation and calling a method without parentheses' do
```

Comment:
string concatenation -> line continuation (or you can just remove this, as it doesn't seem very relevant)

Reply:
Sure! Indeed this is "line continuation" 💦 I've updated it. (It can be removed, but it remains for the testing information)